classes/Hooks.php

```diff
@@ -132,6 +134,23 @@ public static function onBeforeInitialize(&$title, &$article, &$output, &$user,
 		}
 	}
 
+    /**
+     * @param OutputPage $output
+     * @param Skin $skin
+     *
+     * @return void
+     */
+    public static function onBeforePageDisplay( $output, $skin ) {
+        if (
+            self::$profilePage instanceof \CurseProfile\ProfilePage
+            && self::$profilePage->isProfilePage()
```

Comment:
Are you sure this check is enough to make sure these assets are loaded only on user page?

Reply:
Yep, this is the same check as here: https://github.com/Wikia/mediawiki-extensions-CurseProfile/blob/master/classes/Hooks.php#L626